docs/advanced_guide/cloud/notebooks/text_generation/text_generation.ipynb

```diff
@@ -0,0 +1,469 @@
+        "\n",
+        "We provision a Linux [compute cluster](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-create-attach-compute-cluster?tabs=python) fos the NAS job in this Notebook. See the [full list](https://azure.microsoft.com/en-ca/pricing/details/machine-learning/) on VM sizes and prices."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "cpu_compute_name = \"nas-cpu-cluster-D14-v2\"\n",
+        "aml_helper.create_compute_cluster(ml_client, cpu_compute_name, size=\"Standard_D14_v2\")"
+      ]
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "### Create a GPU compute cluster\n",
+        "\n",
+        "For full training we provision a GPU compute cluster."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "gpu_compute_name = \"nas-gpu-cluster-NC6\"\n",
+        "aml_helper.create_compute_cluster(ml_client, gpu_compute_name, size=\"Standard_NC6\")"
+      ]
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "### Create an environment based on a YAML file"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "archai_job_env = aml_helper.create_environment_from_file(ml_client, conda_file=\"conda.yaml\")"
+      ]
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "source": [
+        "### Job 1: NAS (Searching for Pareto-optimal Architectures)"
+      ]
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "source": [
+        "#### Load the search job from a YAML file and run it"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "gather": {
+          "logged": 1675948736346
+        },
+        "jupyter": {
+          "outputs_hidden": false,
+          "source_hidden": false
+        },
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "outputs": [],
+      "source": [
+        "search_job = load_job(source=os.path.join(\"src\", \"search.yaml\"))\n",
+        "s_job = ml_client.create_or_update(search_job)"
+      ]
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "#### Stream logs of the job"
```

Comment:
a bit more detail here would be nice, like "This cell of your notebook will automatically complete when the job is finished".

Reply:
Done.